piker/config.py

```diff
@@ -212,6 +217,10 @@ def load(
         # if one exists.
         if os.path.isfile(template):
             shutil.copyfile(template, path)
+        else:
+            # create an empty file
+            with open(path, 'x'):
```

Comment:
The other thing i was going to maybe suggest is just use `pathlib` here as well to ensure the file exists?

Not a big deal, we can always do it later if needed.

Reply:
Which file exists? We are creating an empty file in this case, the file would def not exist.